tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/indexed_events.yaml

```diff
@@ -0,0 +1,143 @@
+_file_name: indexed_events.yaml
```

Comment:
Are we using this?

Reply:
Yes, to write to the file, we could use the original file name really

Reply:
Sorry, I meant the whole indexed_events.yaml dataset

Reply:
Yes, should we move it to dbt or improve the report?